Library/Homebrew/utils/github.rb

```diff
@@ -659,7 +660,9 @@ def self.check_for_duplicate_pull_requests(name, tap_remote_repo, file:, quiet:
       Manually open these PRs if you are sure that they are not duplicates (and tell us that in the PR).
     EOS
 
-    if !official_tap
+    if strict
+      raise duplicates_message
```

Comment:
To match what happens in the other failure cases:

```suggestion
      odie <<~EOS
        #{duplicates_message.chomp}
        #{error_message}
      EOS
```

Although, the whole statement will probably need to be updated to something like this to appease `rubocop`: (not tested)

```ruby
if strict || (version && official_tap)
  odie <<~EOS
    #{duplicates_message.chomp}
    #{error_message}
  EOS
elsif !official_tap
  opoo duplicates_message
elsif quiet
  opoo error_message
else
  opoo <<~EOS
    #{duplicates_message.chomp}
    #{error_message}
  EOS
end
```